### PR TITLE
Adding route permissons

### PR DESCRIPTION
**Pull request by cesar82**

### CODE REVIEW

File: src/components/templates/PrivateTemplate/index.js [hunk 1]
```diff
@@ -8,11 +8,21 @@ import { I18nContext } from 'i18n';
...
   const user = reactLocalStorage.getObject('user');
   if (isEmpty(user)) navigate('/');
-  const { component: Component, bread } = props;
+
+  // Validate page permissons
+  if (permissons && !checkPermission(permissons)){
+    setTimeout(() => {
+      navigate('/home')
+    });
+  }
```
Comment (normandanielmq) [inline src/components/templates/PrivateTemplate/index.js:24]: Could we do this as a HOC so we can reuse this logic on any component?

Comment (fabian202) [reply src/components/templates/PrivateTemplate/index.js:24]: I think it can be a custom hook instead of an HOC

Comment (cesar82) [reply src/components/templates/PrivateTemplate/index.js:24]: agreed, I like more a custom hook

Comment (normandanielmq) [reply src/components/templates/PrivateTemplate/index.js:24]: As long as we totally prevent the render of the component (blink effect) I am good with that approach.

Comment (cesar82) [reply src/components/templates/PrivateTemplate/index.js:24]: rethinking, we already have utils/permissions.js we can add HOC there for this

**Review by fabian202 — APPROVED** [review]

**Pull request merged**